generators/server/templates/src/main/resources/swagger/api.yml.ejs

```diff
@@ -35,7 +35,8 @@ components:
       content:
         "application/problem+json":
           schema:
-            $ref: 'https://opensource.zalando.com/problem/schema.yaml#/Problem'
+            $ref: 'https://tools.ietf.org/html/rfc7807'
+
```

Comment:
This won't work, has the current reference is to a problem object structure available in https://opensource.zalando.com/problem/schema.yaml#/Problem
We need to check if spring has created one, or probably we might need to describe our structure in the file